### ISSUE TITLE
selectTextStyle props cannot have Text styles

### ISSUE THREAD

**Issue by varunrayen** [peacechen/react-native-modal-selector#29]
the `selectTextStyle` is supposed to inherit all the `Text` styles, but unfortunately when i use `fontSize` style it says it is invalid

**Comment by mikaello** [comment]
This is duplicate of #25, fixed in PR #26 . Either use current master:

packages.json
```
...,
react-native-modal-selector: "peacechen/react-native-modal-selector",
...,
```

... or wait for `v0.0.20` to be released.

**Comment by peacechen** [comment]
Published v0.0.20
